packages/frames.js/src/render/use-frame.tsx

```diff
@@ -144,7 +193,7 @@ export function useFrame<
       console.error("missing frame");
       return;
     }
-    if (!signerState.hasSigner) {
+    if (!signerState.hasSigner && !dangerousSkipSigning) {
       signerState.onSignerlessFramePress();
       return;
     }
```

Comment:
does this work?

would it not be `if (!signerState.hasSigner && dangerousSkipSigning)`?

Reply:
no its current as is. When `dangerousSkipSigning` is true it shouldn't go into this clause that basically aborts

Reply:
ah got it